src/main/java/org/kainos/ea/data/JobRolesData.java

```diff
@@ -1,36 +1,82 @@
 package org.kainos.ea.data;
 
 import org.kainos.ea.exception.DatabaseConnectionException;
+
 import org.kainos.ea.models.JobRolesResponse;
+import org.kainos.ea.models.JobSpecificationResponse;
 
-import javax.xml.crypto.Data;
-import java.sql.Connection;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.sql.*;
 import java.util.ArrayList;
 import java.util.List;
 
 public class JobRolesData {
 
     //US001 - View Job Roles
     public List<JobRolesResponse> getJobRoles(Connection c) throws SQLException, DatabaseConnectionException {
-        Statement st = c.createStatement();
 
-        ResultSet rs = st.executeQuery(
-                "SELECT * "
-                        + "FROM Role;");
+        String query = "SELECT id, title, capabilityID FROM Role;";
```

Comment:
can you do a join to get the capability title here rather than doing a subsequent query